File: gen.cpp
/*
   ______         
  / ____/__    __ 
 / /  __/ /___/ /_
/ /__/_  __/_  __/
\____//_/   /_/   
     96 characters

Usage:
g++ gen.cpp -o gen
gen [KEY] [SITE]
*/

#include<cstdio>
main(int c,char**v){printf(v[1]);for(int i=0;i<4;++i)putchar(char(v[2][i]^37));}

File: gen.py
'''
    ____        __  __              
   / __ \__  __/ /_/ /_  ____  ____ 
  / /_/ / / / / __/ __ \/ __ \/ __ \
 / ____/ /_/ / /_/ / / / /_/ / / / /
/_/    \__, /\__/_/ /_/\____/_/ /_/ 
      /____/                        
                      100 characters

Usage:
python3 gen.py [KEY] [SITE]
'''

import sys
print(sys.argv[1],end='')
for i in range(0,4):print(chr(ord(sys.argv[2][i])^37),end='')
